src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -1,4 +1,49 @@
 package com.kodilla.ecommercee.controller;
 
+import com.kodilla.ecommercee.dto.UserDto;
+import com.kodilla.ecommercee.exception.UserNotFoundException;
+import com.kodilla.ecommercee.mapper.UserMapper;
+import com.kodilla.ecommercee.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin(origins = "*")
```

Comment:
Dlaczego dodajesz @CrossOrigin teraz?
Nie jest to jakiś błąd. Bardziej przez ciekawość pytam.

Reply:
Jest to potem potrzebne w testach z postmanem, aby aplikacja odpowiadała na zapytania?